### PR TITLE
Updates to Clustering in Practice (Lec07)

### PR DESCRIPTION
**Pull request by sladenheim**
This PR does the following:

- fixes minor typos
- condense RI formula to 1 slide
- re-color 3 cluster figure so colors match (this will avoid confusion that the clustering is not good)
- adds slide comparing RI vs ARIs
- adjusts plots to display color bars
- adjusts plots to compare RI and ARI
- add legend to final MDS embedding figure
- add slide describing k-means++ initialization

### CODE REVIEW

File: ds701_book/07-Clustering-II-in-practice.qmd [hunk 1]
```diff
@@ -18,11 +18,11 @@ __`scikit-learn.`__
 Our goals are to learn:
 
 ::: {.incremental}
-* How clustering is used in practice
-* Tools for evaluating the quality of a clustering
-* Tools for assigning meaning or labels to a cluster
-* Important visualizations
-* A little bit about feature extraction for text
+* How clustering is used in practice.
```
Comment (trgardos) [inline ds701_book/07-Clustering-II-in-practice.qmd:21]: I looked this up. The punctuation rule is that you only need it if it is a complete sentence. Not used for sentence fragments.  My preference is to leave them off fragments.

https://www.grammarly.com/blog/bullet-points/#:~:text=If%20the%20text%20of%20your,need%20to%20end%20with%20punctuation.

File: ds701_book/07-Clustering-II-in-practice.qmd [hunk 2]
```diff
@@ -989,22 +1051,31 @@ labels
...
 ## Recap and Next
 
 We've covered
 
-* How clustering is used in practice
-* Tools for evaluating the quality of a clustering
-* Tools for assigning meaning or labels to a cluster
-* Important visualizations
-* A little bit about feature extraction for text
+* How clustering is used in practice.
```
Comment (trgardos) [inline ds701_book/07-Clustering-II-in-practice.qmd:1072]: sentence fragments, remove punctuation

File: ds701_book/07-Clustering-II-in-practice.qmd [hunk 3]
```diff
@@ -989,22 +1051,31 @@ labels
...
+* Tools for evaluating the quality of a clustering.
+* Tools for assigning meaning or labels to a cluster.
+* Important visualizations.
+* A little bit about feature extraction for text.
 
 Next time, we'll look at 
 
-* Hierarchical clustering
-* Gaussian mixture models
+* hierarchical clustering,
```
Comment (trgardos) [inline ds701_book/07-Clustering-II-in-practice.qmd:1080]: sentence fragments, remove punctuation

File: ds701_book/07-Clustering-II-in-practice.qmd [hunk 4]
```diff
@@ -55,31 +55,32 @@ print("gt_centers: ", gt_centers.shape)
 ::: {.notes}
 `datasets.makeblobs` takes as arguments:
 
-- `n_samples`: the number of samples to generate,
-- `n_features`: the number of features, or in other words the dimensionality of
-each sample
-- `center_box`: the bounds of the cluster centers
-- `random_state`: random seed
-- `return_centers`: We also choose the option to return the centers so we have ground truth
+- `n_samples`: The number of samples to generate.
```
Comment (trgardos) [inline ds701_book/07-Clustering-II-in-practice.qmd:58]: sentence fragments, remove punctuation

**Review by trgardos — CHANGES_REQUESTED** [review]
overall looks good. see my first note about not adding punctuation to sentence fragments in bullet lists. thx.

**Comment by sladenheim** [comment]
Not a problem. I can adjust this. Didn't know about this particular rule. I thought that with the book the punctuation would be better, but I'll revert this.

**Comment by sladenheim** [comment]
Fixed.

**Pull request merged**